dnsdbq.c

```diff
@@ -295,10 +310,11 @@ static const struct verb verbs[] = {
 #define	MAX_KEYS 5
 #define	MAX_JOBS 8
 
-#define CREATE(p, s) if ((p) != NULL) { my_panic("non-NULL ptr"); }	\
-	else if (((p) = malloc(s)) == NULL) { my_panic("malloc failed"); } \
+#define CREATE(p, s) if ((p) != NULL) { my_panic(false, "non-NULL ptr"); } \
+	else if (((p) = malloc(s)) == NULL) { my_panic(true, "malloc"); } \
 	else { memset((p), 0, s); }
 #define DESTROY(p) { if ((p) != NULL) { free(p); (p) = NULL; } }
+#define DEBUG(ge, va) { if (debug_level >= (ge)) debug va; }
```

Comment:
Isn't there a better "new" C way to define such a macro?
#define DEBUG(ge, ...) { if (debug_level >= (ge)) debug__VA_ARGS__; }
though if you change the callers to remove the inner paratheses then a clearer variation can be used:
#define DEBUG(ge, ...) { if (debug_level >= (ge)) debug(__VA_ARGS__); }

Tested with clang and gcc-9 on macOS